### PR TITLE
[rllib] Update multi-gpu impala numbers

### PR DESCRIPTION
**Pull request by ericl**
I was able to get ~3 min runs on Pong to 18/19 reward with a pair of V100s and IMPALA. Unfortunately, it looks like it is not possible to scale throughput further without increasing batch size past 1k, which hurts convergence.

Closes https://github.com/ray-project/ray/issues/3343

### CODE REVIEW

File: python/ray/rllib/agents/impala/impala.py [hunk 1]
```diff
@@ -42,6 +43,8 @@
     "num_parallel_data_loaders": 1,
     # level of queuing for sampling.
     "max_sample_requests_in_flight_per_worker": 2,
+    # max number of workers to broadcast one set of weights to
+    "broadcast_interval": 5,
```
Comment (ericl) [inline python/ray/rllib/agents/impala/impala.py]: In the original benchmark code this was if `random.random()  > 0.8` but making it deterministic is probably better.

**Comment by richardliaw** [comment]
jenkins retest this please

**Review by richardliaw — APPROVED** [review]
Looks OK, although `broadcast_interval` is a bit odd of a name, especially since it doesn't persist over multiple `_steps`

**Comment by ericl** [comment]
Yeah, not sure about the name, but we can always change it later.

**Pull request merged**